karma.conf.js

```diff
@@ -121,36 +120,45 @@ module.exports = function(config) {
         },
         reporters: ['dots'],
         webpack: {
+            mode: 'development',
             devtool: 'inline-source-map',
             externals: {
                 fabric: 'fabric'
             },
             module: {
-                preLoaders: [
+                rules: [
+                    {
+                        test: /\.js$/,
+                        include: /src/,
+                        exclude: /node_modules|bower_components/,
+                        loader: 'eslint-loader',
+                        enforce: 'pre'
+                    },
                     {
                         test: /\.js$/,
                         exclude: /(test|bower_components|node_modules)/,
-                        loader: 'istanbul-instrumenter',
+                        loader: 'istanbul-instrumenter-loader',
                         query: {
                             esModules: true
                         }
                     },
                     {
                         test: /\.js$/,
-                        include: /src/,
-                        exclude: /(bower_components|node_modules)/,
-                        loader: 'eslint-loader'
-                    }
-                ],
-                loaders: [
-                    {
-                        test: /\.js$/,
-                        exclude: /(node_modules|bower_components)/,
-                        loader: 'babel'
-                    },
-                    {
+                        exclude: /node_modules|bower_components/,
+                        loader: 'babel-loader?cacheDirectory',
+                        options: {
+                            babelrc: true
+                        }
+                    }, {
                         test: /\.styl$/,
-                        loader: 'css-loader!stylus-loader?paths=src/css/'
+                        use: [
```

Comment:
A brief version is `use: ['css-loader', 'stylus-loader']`.

Reply:
Thank you.